.env.example

```
@@ -152,14 +152,39 @@ AWS_BEDROCK_REGION=ap-south-1
+AWS_LAMBDA_PROACTIVE_ARN=
+
 # EventBridge (Social/Proactive subagent — scheduled Lambda triggers)
-# Leave empty to use node-cron scheduler instead
+# Leave empty to use node-cron scheduler instead (local dev only)
 AWS_EVENTBRIDGE_RULE_ARN=
 
-# SNS (Phase 2 — squad fan-out notifications)
+# SNS (Social subagent — squad fan-out + cascade notifications)
 AWS_SNS_SQUAD_TOPIC_ARN=
+# SNS (Social subagent — outbound cascade notifications, Issue #93)
+AWS_SNS_NOTIFICATION_TOPIC_ARN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_SNS_NOTIFICATION_TOPIC_ARN key should go before the AWS_SNS_SQUAD_TOPIC_ARN key

(UnorderedKey)